DataFormats/TrackerRecHit2D/src/classes_def.xml

```diff
@@ -131,17 +131,17 @@
 
   // fastsim containers
   
-  <class name="FastTrackerRecHit" ClassVersion="10">
-    <version ClassVersion="10" checksum="1775447631"/>
+  <class name="FastTrackerRecHit" ClassVersion="13">
```

Comment:
Please increase class version by one, not by three or four (all modified classes)

Reply:
changed modified classes to version 11